client/src/templates/Challenges/components/completion-modal.tsx

```diff
@@ -161,7 +159,7 @@ class CompletionModal extends Component<
     } = this.props;
 
     if (isOpen) {
-      executeGA({ event: 'pageview', pagePath: '/completion-modal' });
+      callGA({ event: 'pageview', pagePath: '/completion-modal' });
```

Comment:
I am not 100% sure on why, but if, for example, a Camper submits one of the Front End Development Libraries certification projects, this `callGA` is executed 12 times.

This is with the development build. So, that number should/might be less in production, but I suspect we still only want it to be `1`.

Reply:
Good catch. It's happening because this is called every time the modal renders while `isOpen` is true.

I suspect what we actually want is to call it whenever `isOpen` goes from `false` to `true`.